website/i18n/jp/code.json

```diff
@@ -0,0 +1,362 @@
+    "message": "wgtwoコアは、認証、プロビジョニング、音声、メッセージングおよびデータサービスに関連するフルセットの機能を提供し、お客様のビジネスをトランスフォームさせます。",
+    "description": "Subtitle for the enterprise section"
+  },
+  "home.enterprise.data.description": {
+    "message": "高速でセキュア、かつ地理的な制約がなく、信頼性の高いデータ接続により、エンドユーザーのあらゆるニーズにお応えします。",
+    "description": "Description for the data feature"
+  },
+  "home.enterprise.messaging.description": {
+    "message": "SMS/MMSサービス、ボイスメールの提供。",
+    "description": "Description for the messaging feature"
+  },
+  "home.enterprise.voice.description": {
+    "message": "2G、3G、4G（VoLTE）のキャリアグレードな音声サービスを提供します。",
+    "description": "Description for the voice feature"
+  },
+  "home.enterprise.connectivity.description": {
+    "message": "セキュアな認証とSIMカードのプロビジョニング機能を提供します。",
+    "description": "Description for the connectivity feature"
+  },
+  "home.enterprise.console.description": {
+    "message": "新規契約者のプロビジョニング、新機能の有効化、カスタマーサービスのリクエスト対応を実施します。",
+    "description": "Description for the console feature"
+  },
+  "home.enterprise.support.description": {
+    "message": "24時間365日の3rd line 監視・保守サポート。",
+    "description": "Description for the support feature"
+  },
+  "home.hero.subtitle": {
+    "message": "ボックスは現場にありませんし、EOLも心配なし、多額の先行投資の障壁もありません。wgtwoのクラウドコアは、継続的なイノベーション、破壊的なコスト削減、そして真の柔軟性を提供します。シンプルにお使いいただくためにサービスとして提供され、お客様の成長に合わせて拡張することができます。",
+    "description": "Subtitle for the hero section"
+  },
+  "home.logs.subtitle": {
+    "message": "wgtwoプラットフォームを利用する事業者は、提供する回線を自ら設定することができ、ニーズに合ったイノベーション・プロダクトを自由に追加できる仕組みを得ます。",
+    "description": "Subtitle for the logs section"
+  },
+  "home.quote.title": {
+    "message": "「Working Group Twoとの協業により、当社のコアネットワークをパブリッククラウド上に展開することで、事業をよりシンプルに戻し、運用効率の改善、市場ニーズに基づく新サービスの迅速な導入が可能になることを期待しています。特にMVNO、IoT、プライベートネットワークの分野に重点を置いています。これは、複雑さを軽減しながら、規模と最高レベルのセキュリティを実現することであり、当社のお客様、ひいては当社の市場のエンドユーザーに利益をもたらすものです。\"",
+    "description": "The main quote for the customer quote component"
+  },
+  "home.quote.attribution": {
+    "message": "CKH IODのCEO、Joe Parker氏",
+    "description": "The attribution for the customer quote"
+  },
+  "home.rbe.subtitle": {
+    "message": "通信事業者とエンドユーザーは、ユーザーに価値を提供するエコシステムを備えたモバイルネットワークが必要です。その実現のために、私たちはプログラマブルなモバイルコアネットワークを構築しました。APIをフル実装したプラットフォームを、As-a-Serviceで提供します。そして、通信事業者、ディベロッパー、エンドユーザーが出会い、交流することができるマーケットプレイスを構築することを目指しています。",
+    "description": "Subtitle for the RBE section"
+  },
+  "home.rbe.cloud.description": {
+    "message": "クラウドにモバイルコアを。そして、イノベーションをコアネットワークへ呼び戻します。",
+    "description": "Description for the cloud deployment model"
+  },
+  "home.rbe.api.description": {
+    "message": "wgtwo APIを利用して加入者のプロビジョニングを柔軟に実施",
+    "description": "Description for the API deployment model"
+  },
+  "home.rbe.telco.description": {
+    "message": "セキュアで、開発しやすい、次世代の、そしてサービス豊富な通信プラットフォームで、お客様のビジネスを成功に導きます。End-of-Lifeは過去のものです。",
+    "description": "Description for the telco deployment model"
+  },
+  "theme.ErrorPageContent.title": {
+    "message": "This page crashed.",
```

Comment:
Do we need to add non-translated entries? Will it not fallback to english (default) anyways when it's missing?